<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addstore</class>
 <widget class="QWidget" name="addstore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>商品入库</string>
  </property>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>集成显卡</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_10">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>ABS塑料</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>合金材料</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>商品入库</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>机械键盘</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>薄膜键盘</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>内存容量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>硬盘</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>主题颜色</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>1080P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2K</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4K</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>屏幕刷新率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>A面材质</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>512G SSD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1TB SSG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1TB HDD</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>屏幕尺寸</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>13.3英寸</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14.0英寸</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15.6英寸</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17.3英寸</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>入库</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>4G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>64G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>银白色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>水蓝色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>红黑色</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>粉红色</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>键盘特性</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>屏幕分辨率</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CPU</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>60Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>144Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>240Hz</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Intel i5</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <item>
    <property name="text">
     <string>Intel i5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Intel i7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Intel i9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AMD R5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AMD R7</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>GPU显存容量</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
